bin/magi.rs

```diff
@@ -68,13 +51,17 @@ pub struct Cli {
     #[clap(long)]
     jwt_secret: Option<String>,
     #[clap(short = 'v', long)]
-    rpc_port: Option<u16>,
-    #[clap(short = 'p', long)]
     verbose: bool,
+    #[clap(short = 'p', long)]
+    rpc_port: Option<u16>,
     #[clap(long)]
     logs_dir: Option<String>,
     #[clap(long)]
     logs_rotation: Option<String>,
+    #[clap(long)]
+    checkpoint_hash: Option<String>,
+    #[clap(long)]
+    l2_trusted_rpc_url: Option<String>,
```

Comment:
Can we rename this to `checkpoint_sync_url`? Same with the config and docker setup.

Usually consensus clients follow that naming convention, and I also think it helps with making it clear that this flag is only needed when when using checkpoint sync.

Reply:
sure